RenderSystems/GLES2/src/OgreGLES2RenderSystem.cpp

```diff
@@ -844,6 +844,23 @@ namespace Ogre {
 
         GLenum target = mTextureTypes[unit];
 
+        if (target == GL_TEXTURE_EXTERNAL_OES)
```

Comment:
I do not clearly understand what I should do :(

Reply:
at the linked line:
```cpp
if (texPtr->getTextureType() == GL_TEXTURE_EXTERNAL_OES || texPtr->getTextureType() == TEX_TYPE_2D_RECT)
    setTextureAddressingMode( TAM_CLAMP );
else ..
```
instead of the check here